config/locales/en.yml

```diff
@@ -547,6 +547,8 @@ en:
       submitted_by_team: "Submitted by team "
     CoursesReview__SubmissionInfoCard:
       submission_hash: "Submission #"
+    CoursesStudents__TeamsList:
+      last_seen: "| Last seen: "
```

Comment:
Will be good to consider passing the time as variable `Last seen: #{time_string}` and pass `time_string` as variable to the translation method `t`

Let's also maybe remove character `|` from the string and consider using element border . Have asked @irajsuhail to have a UI check here.